CHANGELOG.md

```diff
@@ -2,13 +2,27 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+- Renamed `Improbable.Gdk.Core.EntityQuerySnapshot` to `Improbable.Gdk.Core.EntitySnapshot`. The `Improbable.Gdk.Core.Commands.WorldCommands.EntityQuery.ReceivedResponse` has been updated accordingly. [#1053](https://github.com/spatialos/gdk-for-unity/pull/1053)
+- `DynamicSnapshot` and `DynamicConverter` have been collapsed into `Dynamic`. [#1053](https://github.com/spatialos/gdk-for-unity/pull/1053)
+    - The delegates from all of these classes are now available in the `Dynamic.VTable<TData, TUpdate, TSnapshot>` struct.
+    - `Dynamic.IHandler.Accept` now takes a `Dynamic.Vtable<TData, TUpdate, TSnapshot` parameter.
+
 ### Added
 
-- Added a `TryGetComponent<T>(out T component);` method to the `SpatialOSEntity` struct. This can help reduce boilerplate when writing custom `IEntityGameObjectCreator` implementations. [#1049](https://github.com/spatialos/gdk-for-unity/pull/1049)
+- Added a `bool TryGetComponent<T>(out T component);` method to the `SpatialOSEntity` struct. This can help reduce boilerplate when writing custom `IEntityGameObjectCreator` implementations. [#1049](https://github.com/spatialos/gdk-for-unity/pull/1049).
```

Comment:
```suggestion
- Added a `bool TryGetComponent<T>(out T component);` method to the `SpatialOSEntity` struct. This can help reduce boilerplate when writing custom `IEntityGameObjectCreator` implementations. [#1049](https://github.com/spatialos/gdk-for-unity/pull/1049)
```